cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -54,12 +56,16 @@ if(ANDROID)
   # * https://wiki.openssl.org/index.php/Android#Build_the_OpenSSL_Library
   # Set environment variables similar to 'setenv-android.sh' script:
   # * https://wiki.openssl.org/index.php/Android#Adjust_the_Cross-Compile_Script
-  set(configure_command
-      # Ignored. Prevents script from checking host uname
-      RELEASE=2.6.37
-      SYSTEM=android
-      ARCH=${ANDROID_SSL_ARCH}
-      ${configure_command})
+  # Using the documented method is currently broken on Clang
```

Comment:
I just changed ./Configure back to ./config without changing anything else I get:
***** Unsupported options: no-canonical-prefixes

With ./Configure, I also see -no-canonical-prefixes correctly placed in the arguments to gcc/clang.

There is no other way this script can be written. I am not sure what you are wanting here.